### PR TITLE
pull request template

### PR DESCRIPTION
**Pull request by simbilod**
@tcaduser we can add a pull request template like so to automatize the licensing procedure of contributions (feel free to change the text as you see fit)

### CODE REVIEW

**Comment by simbilod** [comment]
this is associated with the discussion in #136

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+*Description of changes:*
+
+Describe your changes here, linking to an issue if available.
+
+*Licensing*
+
+To maintain the integrity of the project, we require contributions to be licensed under the Apache License Version 2.0. Please acknowledge this by checking the following check-box. By checking the box, you also acknowledge that you have the authority to license your contribution in this manner. You should also ensure that any new file you add contains "# SPDX-License-Identifier: Apache-2.0" in the header.
+
+ - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2).
```
Comment (tcaduser) [inline .github/pull_request_template.md:11]: looks pretty good.

Please:
* change the hyper link to `https://apache.org/licenses/LICENSE-2.0`
* *licenced* to **licensed**.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+
+*Description of changes:*
+
+Describe your changes here, linking to an issue if available.
+
+*Licensing*
+
+To maintain the integrity of the project, we require contributions to be licensed under the Apache License Version 2.0. Please acknowledge this by checking the following check-box. By checking the box, you also acknowledge that you have the authority to license your contribution in this manner. You should also ensure that any new file you add contains "# SPDX-License-Identifier: Apache-2.0" in the header.
```
Comment (tcaduser) [inline .github/pull_request_template.md:9]: Please change:
* Please acknowledge this by checking the following check-box.*
** Please acknowledge this by checking the box below .**

**Comment by tcaduser** [comment]
Please move this to the root directory of the project, as this is acceptable according to GitHub.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

File: .github/pull_request_template.md (hunk 1 above)
Comment (simbilod) [reply .github/pull_request_template.md:11]: done!

File: .github/pull_request_template.md (hunk 2 above)
Comment (simbilod) [reply .github/pull_request_template.md:9]: done!

**Comment by simbilod** [comment]
> Please move this to the root directory of the project, as this is acceptable according to GitHub.
> 
> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

done!

**Comment by tcaduser** [comment]
thank you @simbilod

**Comment by tcaduser** [comment]
It looks like I already had the .github directory in there so I moved the file in there.  sorry for the confusion.

**Comment by tcaduser** [comment]
I confirmed the template works by cherry-picking and placing into main branch

**Pull request merged**